packages/objectloader/src/index.js

```diff
@@ -253,7 +253,7 @@ export default class ObjectLoader {
       count += 1
       yield obj
     }
-    console.log(`Loaded ${count} objects in: ${(Date.now() - t0) / 1000}`)
+    // console.log(`Loaded ${count} objects in: ${(Date.now() - t0) / 1000}`)
```

Comment:
I'd comment these lines out too to make the linter happy, and we're good to merge! 
```
const t0 = Date.now()
let count = 0
```
Happy to keep them as comments as we will be benchmarking loading times on this in the future 😅